epochStart/metachain/economics_test.go

```diff
@@ -269,6 +301,7 @@ func TestEconomics_ComputeEndOfEpochEconomics(t *testing.T) {
 func TestEconomics_VerifyRewardsPerBlock_DifferentHitRates(t *testing.T) {
 	t.Parallel()
 
+	commAddress := "communityAddress"
```

Comment:
there should be a problem with the test above, line 288 and 289. It should have 2 shards and 1 metachain,but there are lastFinalizedHeaders on shard 1 and 2. The number of blocks in the previous epoch is wrongly computed.

Reply:
the function computes the nonces from last finalized headers, and indeed the shards are not correctly set.

I think the idea was that the shards did not create any block in the previous epoch, only the metachain the start of epoch block, so the number of blocks = 1, which is correct even with invalid shard IDs (anyway the last notarized blocks from those shards are behind the first block in epoch)

Reply:
yes. it's ok right now